### ISSUE TITLE
contenful space environment use is not used by export

### ISSUE THREAD

**Issue by roseeng** [contentful/contentful-cli#117]
I can select a default space using ```contentful space use```, and a default environment using ```contentful space environment use```.
If I perform a ```contentful space export``` command after that, without supplying space-id or environment-id, the previously stored space is used, but for environment it will use "master".

## Expected Behavior

```contentful space export``` without any supplied environment-id should use the environment previously stored with ```contentful space environment use```

## Actual Behavior

If no environment-id is supplied to ```contentful space export``` it will use "master".

## Steps to Reproduce

1. Create a second environment in you space. Make sure it contains different data than "master".
2. ```contentful space use <your space-id>```
3. ```contentful space environment use <your envirnoment-id>```
4. ```contentful space export```

NOTE: The order of the use commands is important. If you set the space *after* you have set the environment, environment gets reset to "master". This is by design and as documented.

If you check the exported data, it will come from master, when it should have come from "your environment-id".

## Context

This is not a big problem, I have just started to explicitly supply environment-id. However, it really puzzled me in the beginning, since "master" and my target environment were very similar.

## Environment

- Node Version: v10.6.0
- Npm Version: 6.1.0
- Operating System: Windows 10 64-bit 
- contentful-cli version: 0.18.0

**Comment by roseeng** [comment]
FWIW, I notice that on line 20 in lib/cmds/space_cmds/export.js, the argument is declared as:

```
    .option('environment-id', {
      describe: 'ID of Environment with source data',
      type: 'string',
      default: 'master'
    })
```

My guess is that a missing environment-id argument gets the default value of "master", and later in the code, the export logic assumes that this has been supplied on the command line and therefore should override the previously stored value from the "use"-command.

**Comment by roseeng** [comment]
This is fixed in Release [0.20.0](https://github.com/contentful/contentful-cli/releases/tag/v0.20.0)
